.github/actions/backport/action.yml

```diff
@@ -0,0 +1,73 @@
+name: Backport
+
+inputs:
+  GITHUB_TOKEN:
+    required: true
+  REPO_LOCATION:
+    required: true
+  TARGET_BRANCH:
+    required: true
+  COMMIT_SHA:
+    required: true
+  BACKPORT_OPTIONS:
+    required: false
+
+env:
+  # Not possible to set this as a default
+  # https://github.com/orgs/community/discussions/46670
+  shell: bash
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: hazelcast/backport
+        # Hardcoded as composite actions don't support env vars properly
+        # https://github.com/orgs/community/discussions/51280
+        path: backport
```

Comment:
Maybe it's possible to use `${GITHUB_ACTION_PATH}/backport` and avoid this checkout and polluting the workspace? Also, maybe this way you won't have to introduce `REPO_LOCATION`

Reply:
> Maybe it's possible to use `${GITHUB_ACTION_PATH}/backport` and avoid this checkout and polluting the workspace? Also, maybe this way you won't have to introduce `REPO_LOCATION`

Feels a bit sneaky but works, thanks.

https://github.com/hazelcast/backport/pull/17/commits/77557243fa481e00351ba426b7b17ef3a541d576

Reply:
And removed `REPO_LOCATION` - https://github.com/hazelcast/backport/pull/17/commits/17853e2d5636f5853e3efd71427acec7fecab464